crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4853,3 +4854,167 @@ impl From<AdyenStatus> for storage_enums::PayoutStatus {
         }
     }
 }
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenAcceptDisputeRequest {
+    dispute_psp_reference: String,
+    merchant_account_code: String,
+}
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenMetaData {
+    merchant_account_code: String,
+}
+
+impl TryFrom<(&types::AcceptDisputeRouterData, String)> for AdyenAcceptDisputeRequest {
+    type Error = Error;
+    fn try_from(data: (&types::AcceptDisputeRouterData, String)) -> Result<Self, Self::Error> {
+        let (item, merchant_account_code) = data;
+        Ok(Self {
+            dispute_psp_reference: item.clone().request.connector_dispute_id,
+            merchant_account_code,
+        })
+    }
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenDefendDisputeRequest {
+    dispute_psp_reference: String,
+    merchant_account_code: String,
+    defense_reason_code: String,
+}
+
+impl TryFrom<(&types::DefendDisputeRouterData, String)> for AdyenDefendDisputeRequest {
+    type Error = Error;
+    fn try_from(data: (&types::DefendDisputeRouterData, String)) -> Result<Self, Self::Error> {
+        let (item, merchant_account_code) = data.clone();
+        Ok(Self {
+            dispute_psp_reference: item.request.connector_dispute_id.clone(),
+            merchant_account_code,
+            defense_reason_code: "SupplyDefenseMaterial".into(),
```

Comment:
Can you please attach the screenshot of adyen support confirming the usage of `SupplyDefenseMaterial` in all Defend dispute requests?

Reply:
<img width="825" alt="image" src="https://github.com/user-attachments/assets/3eb2c4cd-3ec0-47a3-8938-e07550072f57">

Reply:
What about the production environment?

Reply:
Yes, We can use this for production also